content/chronograf/v1.8/administration/creating-connections.md

```diff
@@ -23,20 +23,54 @@ To create an InfluxDB connection in the Chronograf UI:
-3. Enter values for the following fields:
 
-    <img src="/img/chronograf/1-7-influxdb-connection-config.png" style="width:100%; max-width:600px;">
-
-    * **Connection URL**: Enter the hostname or IP address of the InfluxDB instance and the port. The field is prefilled with  `http://localhost:8086`.
-    * **Connection Name**: Enter the name for this connection.
-    * **Username**: Enter the username that will be shared for this connection.
-      *Required only if [authorization is enabled](/{{< latest "influxdb" "v1" >}}/administration/authentication_and_authorization/) on the InfluxDB instance to which you're connecting.*
-    * **Password**: Enter the password.
-      *Required only if [authorization is enabled](/{{< latest "influxdb" "v1" >}}/administration/authentication_and_authorization/) on the InfluxDB instance to which you're connecting.*
-    * **Telegraf Database Name**: This field specifies the database that Chronograf will use for populating different parts of the application, including the Host List page. If none is provided, we will use `autogen`. You will still be able to query any database you have access to in the InfluxDB instance when building dashboards or exploring data.
-    * **Default Retention Policy**: Enter the name of the default [retention policy](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp). If none is provided, it assumes `autogen`. If you've changed the default retention policy in your InfluxDB instance, you may want to change it here as well.
-    * **Make this the default connection**: When this option is selected, this InfluxDB connection will be used when Chronograf is launched.
+    ![Chronograf connections landing page](/img/chronograf/1-6-connection-landing-page.png)
+
+3. Provide the necessary connection credentials.
+
+    {{< tabs-wrapper >}}
+    {{% tabs %}}
+[InfluxDB 1.x](#)
+[InfluxDB Cloud or OSS 2.x ](#)
+    {{% /tabs %}}
+    {{% tab-content %}}
+<img src="/img/chronograf/1-8-influxdb-v1-connection-config.png" style="width:100%; max-width:798px;"/>
+
+- **Connection URL**: hostname or IP address and port of the InfluxDB 1.x instance
+- **Connection Name**: Unique name for this connection.
+- **Username**: InfluxDB 1.x username
+  _(Required only if [authorization is enabled](/{{< latest "influxdb" "v1" >}}/administration/authentication_and_authorization/) in InfluxDB)_
+- **Password**: InfluxDB password
+  _(Required only if [authorization is enabled](/{{< latest "influxdb" "v1" >}}/administration/authentication_and_authorization/) in InfluxDB)_
+- **Telegraf Database Name**: the database Chronograf uses to populate parts of the application, including the Host List page (default is `autogen`)
+- **Default Retention Policy**: default [retention policy](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp) (default is `autogen`)
+- **Default connection**: use this connection as the default connection
```

Comment:
use this connection as the default connection for data exploration, dashboard and other administrative actions